ext/standard/config.m4

```diff
@@ -53,6 +53,7 @@ fi
 
 PHP_CHECK_FUNC(crypt, crypt)
 PHP_CHECK_FUNC(crypt_r, crypt)
+AC_CHECK_HEADERS(crypt.h)
```

Comment:
configure.ac already checks this

Reply:
was missing for me.... probably because I run "phpize; ./configure" in the extension directory to "quick" test

Reply:
removed